static/app/utils/profiling/gl/utils.ts

```diff
@@ -14,6 +17,22 @@ import {SpanChartRenderer2D} from '../renderers/spansRenderer';
 import {SpanChartNode} from '../spanChart';
 import {Rect} from '../speedscope';
 
+export function initializeFlamegraphRenderer(
+  renderers: FlamegraphRendererConstructor[],
+  constructorArgs: ConstructorParameters<FlamegraphRendererConstructor>
+): FlamegraphRenderer | null {
+  for (const renderer of renderers) {
+    const r = new renderer(...constructorArgs);
```

Comment:
Won't this throw in some cases and never try to next renderer class?

Reply:
Hmm, it's possible. Let me wrap it in a try/catch